src/docs/implementations/dynatrace.adoc

```diff
@@ -4,25 +4,190 @@ Jon Schneider <jschneider@pivotal.io>
 :sectnums:
 :system: dynatrace
 
-Dynatrace is a dimensional time-series SaaS with built-in dashboarding.
+https://www.dynatrace.com/[*Dynatrace*] is a Software Intelligence Platform featuring application performance monitoring (APM), artificial intelligence for operations (AIOps), IT infrastructure monitoring, digital experience management (DEM), and digital business analytics capabilities.
+It can ingest multi-purpose dimensional time-series data and has built-in dashboarding. Both SaaS and self-hosted (Managed) deployments are offered.
 
 include::install.adoc[]
 
 == Configuring
 
+For setting up new integrations with Dynatrace, it is recommended to use the latest version of the https://www.dynatrace.com/support/help/dynatrace-api/environment-api/metric-v2/[Dynatrace Metrics API] (`api-version: v2`).
+More details on how to use the v2 API are provided further <<bookmark-apiv2, below>>.
+
+
+=== Using a Dynatrace OneAgent installed on the host (preferred) [[bookmark-oneagent-example]]
+
+If a Dynatrace OneAgent is installed on the host running Micrometer, metrics can be exported directly using the OneAgent without having to specify an endpoint URI or API token.
+Use the following code in your project to export Micrometer metrics to the https://www.dynatrace.com/support/help/how-to-use-dynatrace/metrics/metric-ingestion/ingestion-methods/local-api/[local OneAgent endpoint]:
+
 [source,java]
 ----
 DynatraceConfig dynatraceConfig = new DynatraceConfig() {
+    @Override
+    public DynatraceApiVersion apiVersion() {
+        return DynatraceApiVersion.V2;
+    }
+
+    @Override
+    @Nullable
+    public String get(String k) {
+        // This method for retrieving arbitrary config items introduced by the interface is currently not used in DynatraceConfig.
+        return null;
+    }
+};
+MeterRegistry registry = new DynatraceMeterRegistry(dynatraceConfig, Clock.SYSTEM);
+----
+
+If you are working with Spring Boot, you can instead put the following properties in your `application-dynatrace.yml` in the `resources` folder.
```

Comment:
I'm not sure `application-dynatrace.yml` is the right way to do this, I would just say add the right properties:
>DynatraceConfig is an interface with a set of default methods. If, in the implementation of get(String k), rather than returning null, you instead bind it to a property source, you can override the default configuration. For example, Micrometer’s Spring Boot support binds properties prefixed with `management.metrics.export.dynatrace` directly to the DynatraceConfig:

```
management.metrics.export.dynatrace:
    api-version: v2 # Which API do you want to use V1 or V2. (Default: V1)
```